### PR TITLE
Router

### PR DESCRIPTION
**Pull request by cvega08**
##Summary

I used vue-router to change of sign-up route to login route.
Application loads on the route / then redirects to / sign_up

##Screen Shot 
router

##Trello Card
https://trello.com/c/Y8iP9XNx

### CODE REVIEW

File: vue/wolox-books/src/views/Register.vue [hunk 1]
```diff
@@ -72,6 +83,9 @@ export default {
           }
         }
       }
+    },
+    goLogin () {
+      this.$router.push('/login')
```
Comment (FrankIglesias) [inline vue/wolox-books/src/views/Register.vue]: Maybe you'd like to save the token here

File: vue/wolox-books/src/views/Register.vue [hunk 2]
```diff
@@ -13,10 +13,14 @@
           input.input-text-content(:id='field.name' v-model='user[field.name]' )
           p.field-error(v-show='$v.user[field.name].$error')
             |{{ getError($v.user[field.name]) }}
+      p.field-error(v-show='error')
+        |{{ error }}
       button.base-form-button
         |{{ labels.signUp }}
     .container-button
-      button.base-form-button.login-button
+      button.base-form-button.login-button(
```
Comment (FrankIglesias) [inline vue/wolox-books/src/views/Register.vue]: This should be a link

File: vue/wolox-books/src/views/Register.vue (hunk 1 above)
Comment (cvega08) [reply vue/wolox-books/src/views/Register.vue]: Ok, but in this card does not ask save token,  I think is on coming cards

File: vue/wolox-books/src/views/Login.vue [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<template lang="pug">
+  .base-layout-container
+    img.wolox-image(alt='Wolox logo' src='../assets/wolox_logo.svg')
+    .base-form-container
+      h1.base-form-title
+        |Login
```
Comment (FrankIglesias) [inline vue/wolox-books/src/views/Login.vue]: space after pipe

File: vue/wolox-books/src/views/Login.vue [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      button.base-form-button(@click='goSignUp')
+        |Sign up
```
Comment (FrankIglesias) [inline vue/wolox-books/src/views/Login.vue]: same

File: vue/wolox-books/src/views/Login.vue [hunk 5]
```diff
@@ -0,0 +1,31 @@
+<template lang="pug">
+  .base-layout-container
+    img.wolox-image(alt='Wolox logo' src='../assets/wolox_logo.svg')
+    .base-form-container
+      h1.base-form-title
+        |Login
+      button.base-form-button(@click='goSignUp')
```
Comment (FrankIglesias) [inline vue/wolox-books/src/views/Login.vue]: This is a link

File: vue/wolox-books/src/views/Login.vue [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+        |Sign up
+</template>
+
+<script>
+
+export default {
+  name: 'login',
+  methods: {
+    goSignUp () {
+      this.$router.push('/sign_up')
```
Comment (FrankIglesias) [inline vue/wolox-books/src/views/Login.vue]: remove this function

**Pull request merged**